File: app/src/main/java/com/jkb/junbin/sharing/db/DataBaseHelper.java
package com.jkb.junbin.sharing.db;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;
import android.util.Log;

import androidx.annotation.Nullable;

public class DataBaseHelper extends SQLiteOpenHelper {

    public static String message_info = "message_info";
    public static String id = "id";
    public static String content = "content";
    public static String fileName = "fileName";
    public static String date = "date";

    public DataBaseHelper(@Nullable Context context) {
        super(context, "message.db", null, 1);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        createTable(db);
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }

    public void createTable(SQLiteDatabase db) {
        String createTableSql = "CREATE TABLE IF NOT EXISTS " + message_info + "(\n" +
                "\t" + id + " INTEGER PRIMARY KEY AUTOINCREMENT,\n" +
                "\t" + content + " VARCHAR(1024) ,\n" +
                "\t" + fileName + " VARCHAR(1024) ,\n" +
                "\t" + date + " LONG \n" + ")";

        try {
            db.execSQL(createTableSql);
        } catch (Exception e) {
            Log.d("Task:Sql", e.getMessage());
        }
    }
}

File: Feature/message/src/main/java/com/jkb/junbin/sharing/feature/message/db/DataBaseHelper.kt
package com.jkb.junbin.sharing.feature.message.db

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import android.util.Log

class DataBaseHelper(context: Context?) : SQLiteOpenHelper(context, "message.db", null, 1) {
    override fun onCreate(db: SQLiteDatabase) {
        createTable(db)
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {}
    fun createTable(db: SQLiteDatabase) {
        val createTableSql = """CREATE TABLE IF NOT EXISTS $message_info(
	$id INTEGER PRIMARY KEY AUTOINCREMENT,
	$content VARCHAR(1024) ,
	$fileName VARCHAR(1024) ,
	$date LONG 
)"""
        try {
            db.execSQL(createTableSql)
        } catch (e: Exception) {
            Log.d("Task:Sql", e.message!!)
        }
    }

    companion object {
        var message_info = "message_info"
        var id = "id"
        var content = "content"
        var fileName = "fileName"
        var date = "date"
    }
}